src/app/components/mnoe-organizations-list/mnoe-organizations-list.coffee

```diff
@@ -1,32 +1,73 @@
+  link: (scope, elem) ->
 
     # Variables initialization
     scope.organizations =
       search: ''
+      sortAttr: 'created_at'
       nbItems: 10
       page: 1
       pageChangedCb: (nbItems, page) ->
         scope.organizations.nbItems = nbItems
         scope.organizations.page = page
         offset = (page  - 1) * nbItems
-        fetchOrganizations(nbItems, offset)
+        fetchOrganizations(nbItems, offset, scope.organizations.sortAttr)
+
+    $translate(["mnoe_admin_panel.dashboard.organization.account_frozen_state",
+      "mnoe_admin_panel.dashboard.organization.widget.list.table.creation",
+      'mnoe_admin_panel.dashboard.organization.widget.list.table.name'
+      'mnoe_admin_panel.dashboard.organization.widget.list.table.revenue',
+      'mnoe_admin_panel.dashboard.organization.widget.list.table.margin'
+      'mnoe_admin_panel.dashboard.organization.widget.list.table.currency']).then((locale) ->
+        basicFields = [
+          { header: locale['mnoe_admin_panel.dashboard.organization.widget.list.table.name']
+          attr: 'name'
+          skip_natural: true
+          render: (organization) ->
+            template: """
+              <a ui-sref="dashboard.customers.organization({orgId: organization.id})">
+                {{::organization.name}}
+                <em ng-show="organization.account_frozen" class="text-muted" translate>
+                mnoe_admin_panel.dashboard.organization.account_frozen_state</em>
+              </a>
+            """,
+            scope: {organization: organization} }
+          { header: locale["mnoe_admin_panel.dashboard.organization.widget.list.table.creation"],
+          style: {width: '110px'},
+          attr:'created_at',
+          sort_default: "reverse"
+          skip_natural: true
+          render: (organization) ->
+            template:
+              "<span>{{::organization.created_at | date: 'dd/MM/yyyy'}}</span>"
+            scope: {organization: organization}}]
+        scope.organizations.fields = unless MnoeAdminConfig.isFinanceEnabled() then basicFields else basicFields.concat(
+          [{ header: locale['mnoe_admin_panel.dashboard.organization.widget.list.table.revenue'],
+          attr:'financial_metrics.revenue', donotsort: true, style: width: '110px',}
+          { header: locale['mnoe_admin_panel.dashboard.organization.widget.list.table.margin'],
+          attr:'financial_metrics.margin', donotsort: true,  style: width: '110px'}
+          { header: locale['mnoe_admin_panel.dashboard.organization.widget.list.table.currency'],
+          attr:'financial_metrics.currency', donotsort: true, style: width: '110px'}]))
 
-    scope.isFinanceEnabled = MnoeAdminConfig.isFinanceEnabled()
+    # Smart table callback
+    scope.pipe = (tableState) ->
+      # The order has changed - reset pagination
+      scope.organizations.page = 1
+      scope.organizations.sortAttr = tableState.sort.predicate
+      if tableState.sort.reverse
```

Comment:
Indentation seems a bit funny here

Reply:
```
scope.organizations.sortAttr += ".desc" if tableState.sort.reverse
```
?

Reply:
There is also the `fetchOrganizations` in the if.

Reply:
The .desc is there for sorting by descending order, see https://github.com/maestrano/mno-enterprise/tree/4.0/core/app/models/mno_enterprise/base_resource.rb#L39
The `fetchOrganizations` isn't in the `if`.
Would this format:
 ```
if tableState.sort.reverse
    scope.organizations.sortAttr += ".desc"
fetchOrganizations(scope.organizations.nbItems, 0, scope.organizations.sortAttr)
```
be clearer ?